src/components/upload/Upload.vue

```diff
@@ -73,7 +73,8 @@ export default {
         value(value) {
             this.newValue = value
             if (!this.newValue ||
-            (Array.isArray(this.newValue) && this.newValue.length === 0)) {
+            (Array.isArray(this.newValue) && this.newValue.length === 0) ||
+            this.newValue.findIndex((x) => x.name === this.$refs.input.files[0].name) < 0) {
```

Comment:
Please a IE11+ compatible function

Reply:
Ok, I ended up using it because I found the `findIndex` in use in the `Steps.vue` file, in case it is an error I can send a pull in Steps too